archive/common/src/test/scala/uk/ac/wellcome/platform/archive/common/fixtures/RandomThings.scala

```diff
@@ -23,6 +23,14 @@ trait RandomThings {
     Random.alphanumeric take length mkString
   }
 
+  def randomAlphanumericWithSpaces(length: Int = 8) = {
```

Comment:
did I read this right, is it `randomAlphanumericWithSpace`?

Reply:
Yes, it is, will change